chevah/empirical
# [#2746] Fix assertIsEmpty.

Problem
======

assertIsEmpty fails when checking for a non empty tupple

Why we got this
============

no tests
bad practice for string formating not forced in a tupple

Changes
=======

fix and write tests.


How to test
=========

reviewers: @alibotean 

check that changes make sense
version released as needed in compat.

Thanks!
